test/over_react/util/hoc_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+import 'package:test/test.dart';
+
+import '../../over_react_redux/fixtures/counter.dart';
+import '../../over_react_redux/fixtures/store.dart';
+import '../../test_util/test_util.dart';
+
+main() {
+  group('composeHocs', () {
+    test('will render a nested component', () {
+      CounterProps Function() components = composeHocs([
+        connect<CounterState, CounterProps>(),
+        connect<CounterState, CounterProps>(),
```

Comment:
We should test the order these are composed in. You could do something like 
```dart
        connect<CounterState, CounterProps>(
            mapStateToProps: (_) => {
              'data-0': '0',
              'data-both': '0',
            },
        ),
        connect<CounterState, CounterProps>(
            mapStateToProps: (_) => {
              'data-1': '1',
              'data-both': '1',
            },
        ),
```
and verify that `data-both` and the other attributes have the correct values on the rendered node.

Reply:
Good catch!